Wrappers/Python/cil/optimisation/functions/IndicatorBox.py

```diff
@@ -29,67 +34,446 @@ class IndicatorBox(Function):
                                             0, \text{ if } x \in [a, b] \\
                                             \infty, \text{otherwise}
                                      \end{cases}
```

Comment:
The documentation renders incorrectly

![image](https://user-images.githubusercontent.com/60604372/217789931-0d3bcd2f-983f-4ba0-a48a-c657d25a44d6.png)

Reply:
should be fixed now